python/adelphi/bin/adelphi

```diff
@@ -29,25 +34,29 @@ import click
 
 from adelphi.anonymize import anonymize_keyspace
 from adelphi.cql import export_cql_schema
-from adelphi.gemini import export_gemini_schema, format_gemini_schema
+from adelphi.gemini import export_gemini_schema
 from adelphi.gh import build_github, build_origin_repo, build_branch, commit_schemas, build_pull_request
-from adelphi.store import with_cluster, build_keyspace_objects
+from adelphi.store import build_keyspace_objects
+from adelphi.store import with_cluster as _with_cluster
 
 logging.basicConfig(level=logging.INFO)
 log = logging.getLogger('adelphi')
 
+# Note: click assumes a type of string and a required value of False unless explicitly specified
 @click.group()
-@click.option('--hosts', default='127.0.0.1', help='Comma-separated list of contact points. Default: 127.0.0.1')
-@click.option('--port', type=int, default=9042, help='Database RPC port. Default: 9042')
+@click.option('--hosts', default='127.0.0.1', show_default=True, help='Comma-separated list of contact points')
+@click.option('--port', type=int, default=9042, show_default=True, help='Database RPC port')
 @click.option('--username', help='Database username')
 @click.option('--password', help='Database password')
 @click.option('--keyspaces',
               help='Comma-separated list of keyspaces to include. If not specified all non-system keypaces will be included')
 @click.option('--rf', type=int, help='Replication factor to override original setting. Optional.')
-@click.option('--anonymize/--no-anonymize', default=True, help="Enable/disable schema anonymization")
-@click.option('--output-dir', help='Schema files will be written to this directory. If not specified schemas will be written to stdout.')
+@click.option('--anonymize/--no-anonymize', default=True, show_default=True, help="Enable/disable schema anonymization")
+@click.option('--output-dir', help='Directory schema files should be written to. If not specified, it will write to stdout')
+@click.option('--purpose', help='Comments on the anticipated purpose of this schema.  Optional')
+@click.option('--maturity', help='Comments on the maturity of this schema.  Optional')
```

Comment:
I'm wondering if we should offer some guidance here, like list some options in help: alpha, beta, prod, staging, ... I think it gives us a better chance of people actually filing this in and having a consistent format.
I'm not saying we should enforce any of these values, but simply propose them.
@absurdfarce wdyt?

Reply:
Per external conversation we're going to be doing exactly this as a middle ground between a free text field vs. an enum of allowed values.  Change to follow shortly.